README.md

```diff
@@ -1,36 +1,282 @@
+  # production.minify :uglifier # or :yui
+end
+```
+
+## Usage
+
+* `$ hi5 deploy <platform> -e <environment>`
+
+This will build your application for a specific platform with the settings for the given environment to the destination directory specified in `config/high_five.rb`.
+
+* There's a set icon task `$ hi5 <platform>:set_icon path/to/icon.png` I think?.
+
+TODO: Is it really <platform> or just android/ios?
+
+By default, high_five uses ChunkyPNG to resize the icon, but this almost always creates shitty looking icons.
+We highly recommend adding the [rmagick](https://github.com/rmagick/rmagick) gem (which depends on imagemagick) to your Gemfile along with high five.
+High Five will detect that you have rmagick installed and will use it to generate much nicer icons.
+
+TODO: Just make rmagick a dependency since ChunkyPNG is shit and no one will want to use it anyways?
+
+* Pretty sure there's a task to set version? But I don't really know what that means, what it does, how it works, or how to run it.
+
+* There's a dist command that we should document. It has an --install option that's really freaking useful. Not sure what dist does without --install.
+
+* There's probably more high five commands that should be documented
+
+### Development Usage
+
+#### Build and deploy for browser
+
+`hi5 deploy web`
+
+Then open index-debug.html inside of google chrome with web security disabled (`google-chrome --disable-web-security`)
+
+TODO: Can't we just open www/index.html instead? WHY THE EXTRA FILE??
+
+#### Build and deploy for device
+
+Make sure your device and computer are on the same wifi network (subnet).
+
+Make sure your server accepts connects from devices on the same subnet.
+
+e.g. For rails, start your server like this:
+
+`rails server -b 0.0.0.0`
+
+##### Android
+
+According to Brett, only the _first_ time you put a build on your device, you need to do this (replace the keystore, alias, password, and alias password with the correct values for your app):
+
+`hi5 deploy android --environment lan && hi5 dist android --environment lan --install --ant -o "Looper-android--adhoc" -e --ant-flags="-Dkey.store=looper.keystore -Dkey.alias=looper -Dkey.store.password=looper -Dkey.alias.password=looper"`
+
+TODO: Write some words on how to find those options (set by the person who setup the project initially?)
+
+TODO: What is the "-o" option? Use the long version in documentation for clarity.
+
+TODO: What is the "-e" option? Is that an ant option? Is it different from the deploy -e (environment) option? If it is, turn this TODO into a NB so that the person reading this readme understands that (because it's super confusing).
+Using the long version should make that more clear.
+
+After that, this should be sufficient:
+
+`hi5 deploy android -e lan && hi5 dist android -e lan --install`
+
+*However*, this did not work for me. I had to always use the first command or else hi5 just kept installing the initial build.
```

Comment:
Actually the reason this doesn't work is because you told it to use ant in the original. if you don't do that it will use gradle and when hi5 uses gradle it creates a file with your keystore stuff in it. It's just a coincidence that it works without the ant keystore flags afterword because that file now exists on your computer.